server/consumer.go

```diff
@@ -32,17 +32,17 @@ import (
 )
 
 type ConsumerInfo struct {
-	Stream         string         `json:"stream_name"`
-	Name           string         `json:"name"`
-	Created        time.Time      `json:"created"`
-	Config         ConsumerConfig `json:"config"`
-	Delivered      SequencePair   `json:"delivered"`
-	AckFloor       SequencePair   `json:"ack_floor"`
-	NumAckPending  int            `json:"num_ack_pending"`
-	NumRedelivered int            `json:"num_redelivered"`
-	NumWaiting     int            `json:"num_waiting"`
-	NumPending     uint64         `json:"num_pending"`
-	Cluster        *ClusterInfo   `json:"cluster,omitempty"`
+	Stream         string          `json:"stream_name"`
+	Name           string          `json:"name"`
+	Created        time.Time       `json:"created"`
+	Config         *ConsumerConfig `json:"config,omitempty"`
```

Comment:
@derekcollison you ok with this? It's so that when asking for 1000s of them via jsz that we can opt out of all the extra data?

Reply:
That particular change was done because some types inside consumer config have a custom json marshaller. Stuff like turning the value 0 into a string.
As a result setting Config = ConsumerConfig{} would result the corresponding values to be printed. 
Therefore Config was never empty and always emitted, even when setting everything to 0.

Reply:
I think that looks ok. I know I did do those like that in some places on purpose to force a copy and not hold onto what the user gave us. Let's make sure that semantic is preserved.

Reply:
Fixed. Made sure that we operate on a copy as before

Reply:
Thx, if R.I. LGTM its good by me.